### DISCUSSION TITLE
404 when using o1-preview model

### DISCUSSION THREAD

**Discussion by lucaseatp** [langgenius/dify#8335] [Help] [answered]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.8.1

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

update to 0.8.1 and try to use o1-preview

### ✔️ Expected Behavior

o1-preview to run normally

### ❌ Actual Behavior

Run failed: [openai] Bad Request Error, Error code: 404 - {'error': {'message': 'The model o1-preview does not exist or you do not have access to it.', 'type': 'invalid_request_error', 'param': None, 'code': 'model_not_found'}}

**Comment by laipz8200** [comment, marked as answer]
If you're using your own API key, please note that only users with Tier 5 or higher have access to this model.

Here is a document: https://platform.openai.com/docs/guides/reasoning
